### ISSUE TITLE
Bug: Previewing Office files in the info pane causes crash

### ISSUE THREAD

**Issue by CAPSLOOCK** [files-community/Files#15505]
### Description

The app is crashing or freezing every time i'm opening the preview pannel for a .docx/.xlsx/.pptx file. 
For some other types of files, the app doesn't crash but nothing is displayed, e.g.  for .pdf files the app seems to be running forever, without showing anything. 

### Steps To Reproduce

1. Open the app for the first time after launch
2. Select a file (e.g. .docx/.xlsx/.pptx file in my case)
3. Open the preview panel
4. Try to see the preview

### Requirements

- Fix the preview pannel with .docx/.xlsx/.pptx files to prevent the app from crashing. 
- Fix the .pdf preview, to make something appear

### Files Version

3.4.4.0

### Windows Version

26120.680 (24H2)

### Log File

[debug.log](https://github.com/files-community/Files/files/15498878/debug.log)

**Comment by yaira2** [comment]
@CAPSLOOCK is this an issue in the stable version?

**Comment by yaira2** [comment]
Fyi @0x5bfa

**Comment by CAPSLOOCK** [comment]
@yaira2 no, in the stable version, everything seems OK for the .xlsx and .docx files. 
But, a freeze occurs when returning to the details panel after viewing the preview for large .pptx files in my case (but this seems to be more to do with the application's performance than a bug). 
Also, nothing is displayed on preview .pdf files either, as in the preview app version.

**Comment by 0x5bfa** [comment]
How big your pptx file is?

And I can’t preview PDF, is there need to external library?

**Comment by Josh65-2201** [comment]
You would need a program to give PDF previews. Power Toys works for me with it.

**Comment by CAPSLOOCK** [comment]
- @0x5bfa After further testing, I noticed that the problem occurs with only large .pptx. And now, i don't have any freezing or crash anymore, the application just seems to load the preview indefinitely, as it does for PDFs. I have had a complete crash only once, with the preview application, as first reported. I made you a video to understand the problem. 
NB: this problem is the **same for the stable app** @yaira2 

https://github.com/files-community/Files/assets/170892796/14650352-08a7-487e-84eb-32158a2a4604


- @0x5bfa I don't understand your question about pdf's, I normally open them with Adobe
 Acrobat without any problems, the problem is that nothing is displayed on the preview panel as you can see.
 ![preview panel-pdf files-stable app](https://github.com/files-community/Files/assets/170892796/76b22a4c-00b0-40e1-90d5-0272fc8b15ce)
- @Josh65-2201 What PowerToys do you use to preview PDFs?

**Comment by 0x5bfa** [comment]
I see. Nvm about my question.
@yaira2 I’m not sure this problem is caused by my change or gave’s.

> with only large .pptx. And

Tell me the size. I’ll try to reproduce

**Comment by CAPSLOOCK** [comment]
@0x5bfa The file is about 30Mb, but something weird happen actually, with the edge of the app, see the video to understand well. I have exactly the same problem with the stable version.

https://github.com/files-community/Files/assets/170892796/fe424fac-8202-4df8-898a-eef21a780429

I don't have that brief appearance with pdf files.

**Comment by 0x5bfa** [comment]
Can you reproduce in docx and xlsx file?

**Comment by yaira2** [comment]
We hope to solve this issue, but it's good to hear this isn't a new regression.

**Comment by yaira2** [comment]
This should be resolved with v4. If it's still an issue after the release, please let us know and we'll take another look.
